### PR TITLE
Change docker image credentials directory to /app from /root

### PR DESCRIPTION
**Pull request by ArturoGuerra**
the matter server requires the credentials folder to be located in the working directory of the server

### CODE REVIEW

**Comment by MartinHjelmare** [comment]
Did you successfully test commissioning with these changes?

**Comment by ArturoGuerra** [comment]
Yeah before it gave an error when trying to commission any device cuz it was unable to find the certs

File: matter_server/server/device_controller.py [hunk 1]
```diff
@@ -68,6 +68,9 @@ async def start(self) -> None:
         self.compressed_fabric_id = await self._call_sdk(
             self.chip_controller.GetCompressedFabricId
         )
+        LOGGER.debug("CHIP Device Controller Initialized")
```
Comment (MartinHjelmare) [inline matter_server/server/device_controller.py:71]: Please remove the non related changes.

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**